src/stripeJavaLanguageClient/javaServerStarter.ts

```diff
@@ -76,23 +78,21 @@ function prepareParams(
     '-Declipse.product=org.eclipse.jdt.ls.core.product',
   );
 
-  if (DEBUG) {
+  if (inDebug) {
     params.push('-Dlog.level=ALL');
   }
 
   const encodingKey = '-Dfile.encoding=';
   params.push(encodingKey + getJavaEncoding());
 
   const serverHome: string = path.resolve(__dirname, javaServerPath);
-  const launchersFound: Array<string> = glob.sync('**/plugins/org.eclipse.equinox.launcher_*.jar', {
-    cwd: serverHome,
-  });
+  const launchersFound: Array<string> = getServerLauncher(serverHome);
 
-  if (launchersFound.length) {
+  if (launchersFound && launchersFound.length) {
     params.push('-jar');
     params.push(path.resolve(serverHome, launchersFound[0]));
   } else {
-    return [];
+   return 'Server jar not found';
```

Comment:
Instead of returning a string what if we throw an exception here? That may make it more clear that something we didn't expect happened and the caller does not have to do a type check. WDYT?

Reply:
Are you suggesting I throw a user defined exception with error message? that sounds much cleaner and i will update it!